assets/syntaxes/Packages/PHP/PHP.sublime-completions

```diff
@@ -2410,7 +2410,7 @@
 		{ "trigger": "newt_label_set_text", "contents": "newt_label_set_text(${1:label}, ${2:text})" },
 		{ "trigger": "newt_listbox", "contents": "newt_listbox(${1:left}, ${2:top}, ${3:height})" },
 		{ "trigger": "newt_listbox_append_entry", "contents": "newt_listbox_append_entry(${1:listbox}, ${2:text}, ${3:data})" },
-		{ "trigger": "newt_listbox_clear", "contents": "newt_listbox_clear(${1:listobx})" },
+		{ "trigger": "newt_listbox_clear", "contents": "newt_listbox_clear(${1:listbox})" },
```

Comment:
At least as of _some_ point in time, I think this is a correct change: http://php.adamharvey.name/manual/en/function.newt-listbox-clear-selection.php (admittedly, some of the time this site includes third party components, e.g. `Inclued`)